templates/travel/elements/_clip.css

```diff
@@ -0,0 +1,59 @@
+/**
+ * clip
+ *
+ * Hide an element without preventing visibility detection.
+ *
+ */
+
+:root .clip {
```

Comment:
namespaces. copyright, and better docs

Reply:
Alright makes sense. Thanks for the feedback!